cluster/gce/trusty/node.yaml

```diff
@@ -262,6 +262,35 @@ end script
 MIME-Version: 1.0
 Content-Type: text/upstart-job; charset="us-ascii"
 Content-Transfer-Encoding: 7bit
+Content-Disposition: attachment; filename="kube-install-additional-packages.conf"
+
+#upstart-job
+
+description "Install additional packages used by kubernetes"
+
+start on stopped kube-docker
+
+script
+	# Socat and nsenter are used in port forwarding. They are not required for spin
+	# up a cluster. Therefore, we move the installation here to make it parallel to
+	# other steps to reduce the cluster creation time.
```

Comment:
I'd take a reliable (but slightly slower) install over a faster flaky install. If the install time needs to be optimized, someone can always build a new image with the tools preinstalled.